CMakeLists.txt

```diff
@@ -854,24 +854,54 @@ set(wrapper_test_scripts
 	test-fake-ssh test-line-buffer test-svn-fe test-tool)
 
 
-foreach(script ${wrapper_scripts})
-	file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
-	string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
-	string(REPLACE "@@PROG@@" "${script}${EXE_EXTENSION}" content "${content}")
-	file(WRITE ${CMAKE_BINARY_DIR}/bin-wrappers/${script} ${content})
-endforeach()
+if(NOT DEFINED CI_BUILD)
```

Comment:
I am still 100% against putting CI-specific blocks into `CMakeLists.txt`.

We really need to introduce knobs that can be asked for explicitly, whether it be in a CI build or because the current developer wants it. We do _not_ want the CI build to differ from a regular build, in general. Otherwise we would test the wrong thing.